### ISSUE TITLE
Add SUBSCRIBE button next to channel name while watching a video.

### ISSUE THREAD

**Issue by qaiserxd** [TeamNewPipe/NewPipe#7277]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?

Add a subscribe button next to the channel name while watching a video like youtube does it.

### NewPipe UI
![IMG_20211019_104504](https://user-images.githubusercontent.com/89571992/137848139-489bab2e-6744-48b4-8081-c2e7c73031be.jpg)

### YouTube UI
![IMG_20211019_104547](https://user-images.githubusercontent.com/89571992/137848327-c7a7c0c3-9c35-436d-83b5-9fcd9083ad78.jpg)

#### Why do you want this feature?

It will be easier to subscribe a channel while watching their content without opening the channel. Right now you have to open their channel to subscribe.

**Comment by u7284728** [comment]
Hi, I would like to work on this. Could someone please point me in the right direction as to which classes/xml files this would be associated with. Thank you :)

**Comment by Stypox** [comment]
The texts should not be so clumped, but at the same time I don't think that that layout should use up more vertical space as it is already using up much.

My idea was to keep views and likes where they are and put the button in between the uploader and the views & likes. But this wouldn't work either, since on small screens and with long channel names you would get a really small space left for the actual channel name:
![image](https://user-images.githubusercontent.com/36421898/197857947-da885291-2056-4d4f-9b3a-5ae23b81caff.png)

Another idea was to use an icon instead of text, in the same place, but there do [not seem to be](https://fonts.google.com/icons?selected=Material+Icons&icon.query=subscribe) any material icon that would be understandable.

**Comment by UIUXGuy** [comment]
My proposal:
Default state [the user is not subscribed yet]
![Proposed  Subscribe](https://user-images.githubusercontent.com/111644438/198575140-d7dd2a25-a1c7-42c9-aa16-0aff71fc61a9.png)
Already subscribed state [when clicked, the user unsubscribes]
![Proposed  Unsubscribe](https://user-images.githubusercontent.com/111644438/198575251-86398848-30e9-4719-b90d-6023ebaf4b0f.png)
Let me know what you think.

**Comment by opusforlife2** [comment]
... not a bad idea, actually. Instead of 5 buttons, though, we could just shift the 'Add To' button down into the 2nd row of buttons.

**Comment by UIUXGuy** [comment]
5-button way is actually common. YouTube itself uses 5-button way in its bottom bar. Facebook went even for 6-button for its navigation. And as Stypox said, more vertical space won't be a good idea, and I totally agree.

**Comment by opusforlife2** [comment]
We already have space for 2 more buttons in the 2nd row. And 5 buttons start to crowd each other on small (4.7") screens.

**Comment by UIUXGuy** [comment]
Correct me if I'm wrong, but I see that we only have 2 rows; the 1st with the channel name, likes, and views, the 2nd with the Dowload, popup, etc. If you meant adding buttons to the 1st, well I don't think it's a good idea since the added buttons could cover the channel's name even more.

**Comment by opusforlife2** [comment]
Tap the video title, which you'll notice has a down arrow at the right.

**Comment by Stypox** [comment]
I like the idea, and I think, as Opus said, that the button should go in the second hidden-by-default row of buttons (subscribing is a pretty rare action)

**Comment by UIUXGuy** [comment]
I think Opus suggested moving "Add To" button to the 2nd row as it's not as primary as "Subscribe" button. I think subscribing is the Call-To-Action in Youtube, that's why it's colored red.

**Comment by Stypox** [comment]
I do not agree but it's not so important, so let's move "Add to playlist" below and put "Subscribe" in the first row
